ACE/include/makeinclude/platform_android.GNU

```diff
@@ -9,7 +9,123 @@ no_hidden_visibility ?= 1
 # as of NDK r6 inlining is required
 inline ?= 1
 
-include $(ACE_ROOT)/include/makeinclude/platform_linux_common.GNU
+# start of: include most of platform_linux_common.GNU
```

Comment:
No need for this comment to my idea, we have git history to know where changes are coming from